### PR TITLE
Update domestic CP minutes for Canada and US

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9226

I confirmed it in my demo tenants:
image
image

And the spreadsheet also contains 3000 minutes for US and Canada:
https://view.officeapps.live.com/op/view.aspx?src=https%3A%2F%2Fdownload.microsoft.com%2Fdownload%2Fd%2Fb%2F6%2Fdb6992c5-ec9a-4b02-8742-59df17dd90a7%2Fcountry-and-region-availability-(v-19OCT2021)-(en-us).xlsx&wdOrigin=BROWSELINK

Also a screenshot here:
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9242

For the UK and all countries in Europe, it seems that the CP have 1200 minutes.

### CODE REVIEW

File: Teams/country-and-region-availability-for-audio-conferencing-and-calling-plans/availability-in-canada.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Here are the Microsoft 365 and Office 365 services for Skype for Business and Mi
...
 |Calling Plans are available to purchase?  |Yes          |   
-|Minutes available for Calling Plans.    |There are five plan options available per calendar month: <ul><li>Pay-As-You-Go Calling Plan </li><li> 120 minutes Domestic Calling Plan </li><li> 240 minutes Domestic Calling Plan </li><li> 1200 minutes Domestic Calling Plan </li><li> International Calling Plan: 1200 minutes of domestic or 600 minutes of international calling (whichever is reached first in the calendar month)</li></ul>|
+|Minutes available for Calling Plans.    |There are five plan options available per calendar month: <ul><li>Pay-As-You-Go Calling Plan </li><li> 120 minutes Domestic Calling Plan </li><li> 240 minutes Domestic Calling Plan </li><li> 3000 minutes Domestic Calling Plan </li><li> International Calling Plan: 3000 minutes of domestic or 600 minutes of international calling (whichever is reached first in the calendar month)</li></ul>|
```
Comment (JohanFreelancer9) [inline Teams/country-and-region-availability-for-audio-conferencing-and-calling-plans/availability-in-canada.md:58]:
```suggestion
|Minutes available for Calling Plans.    |There are five plan options available per calendar month: <ul><li>Pay-As-You-Go Calling Plan </li><li> 120 minutes Domestic Calling Plan </li><li> 240 minutes Domestic Calling Plan </li><li> 3,000 minutes Domestic Calling Plan </li><li> International Calling Plan: 3,000 minutes of domestic or 600 minutes of international calling (whichever is reached first in the calendar month)</li></ul>|
```

File: Teams/country-and-region-availability-for-audio-conferencing-and-calling-plans/availability-in-the-united-states-u-s.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Here are the Microsoft 365 and Office 365 services for Skype for Business and Mi
...
 |Calling Plans are available to purchase?  |Yes          |   
-|Minutes available for Calling Plans.    |There are five plan options available per calendar month: <ul><li> 120 minutes Domestic Calling Plan </li><li> 240 minutes Domestic Calling Plan </li><li> 1200 minutes Domestic Calling Plan </li><li> International Calling Plan: 1200 minutes of domestic or 600 minutes of international calling (whichever is reached first in the calendar month)</li></ul>|
+|Minutes available for Calling Plans.    |There are five plan options available per calendar month: <ul><li> 120 minutes Domestic Calling Plan </li><li> 240 minutes Domestic Calling Plan </li><li> 3000 minutes Domestic Calling Plan </li><li> International Calling Plan: 3000 minutes of domestic or 600 minutes of international calling (whichever is reached first in the calendar month)</li></ul>|
```
Comment (JohanFreelancer9) [inline Teams/country-and-region-availability-for-audio-conferencing-and-calling-plans/availability-in-the-united-states-u-s.md:58]:
```suggestion
|Minutes available for Calling Plans.    |There are five plan options available per calendar month: <ul><li> 120 minutes Domestic Calling Plan </li><li> 240 minutes Domestic Calling Plan </li><li> 3,000 minutes Domestic Calling Plan </li><li> International Calling Plan: 3,000 minutes of domestic or 600 minutes of international calling (whichever is reached first in the calendar month)</li></ul>|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by scanum — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd - Hope everything is well, could you please help us with the copy review? Thanks!

**Pull request merged**